vendor/index.yaml

```diff
@@ -944,6 +944,10 @@ vendors:
     name: Boston Networks Limited
     vendorID: 598
 
+  - id: milesight-iot
+    name: Milesight IoT Co., Ltd.
+    vendorID: 600
```

Comment:
We keep https://lora-alliance.org/wp-content/uploads/2020/11/LoRa_Alliance_Vendor_ID_for_QR_Code.pdf as authoritive index. 600 is reserved for future use.

Were you a LoRa Alliance member before, and was this your ID?